site/gatsby-site/migrations/2022.08.15T19.33.01.remove-commas-in-classifications.js

```diff
@@ -0,0 +1,70 @@
+/** @type {import('umzug').MigrationFn<any>} */
+exports.up = async ({ context: { client } }) => {
+  await client.connect();
+
+  const classificationsCollection = client.db('aiidprod').collection('classifications');
+
+  const classifications = await classificationsCollection.find({});
+
+  while (await classifications.hasNext()) {
+    const classification = await classifications.next();
+
+    if (classification.incident_id == 40) {
+      continue;
+    }
+
+    if (classification.incident_id == 76) {
+      classificationsCollection.updateOne(
+        { incident_id: 76 },
+        {
+          $set: {
+            classifications: {
+              ...classification.classifications,
+              'Data Inputs': [
+                'photo IDs, names birthdays, and national IDs of people suspected of crimes',
+                'camera feed',
+              ],
+            },
+          },
+        }
+      );
+      continue;
+    }
+
+    for (const category of Object.keys(classification.classifications).filter(
+      (category) =>
+        ![
```

Comment:
Can you do the opposite here, so we are sure it is only updating what we want? 

Don't know how it is called, but "positive" filters are safer.